libs/api/ebpf_api.cpp

```diff
@@ -2829,7 +2829,7 @@ _Requires_lock_not_held_(_ebpf_state_mutex) _Must_inspect_result_ ebpf_result_t
         std::string log_message = "ebpf_object_open: error loading file:";
         log_message += path;
         log_message += ", error message:";
-        log_message += *error_message;
+        log_message += *error_message != NULL ? *error_message : "none";
```

Comment:
Might be more helpful to say something like "peparse error message dropped" instead of "none".

Reply:
The source of the error message is not known at this point.